### ISSUE TITLE
Release for purescript 0.14

### ISSUE THREAD

**Issue by darichey** [spicydonuts/purescript-react-basic-hooks#46]
I believe this is blocked by ~~https://github.com/lumihq/purescript-react-basic/issues/135 and~~ ~~https://github.com/garyb/purescript-indexed-monad/issues/31~~. After that, the `ps-0.14` branch can be rebased and merged.

**Comment by milesfrain** [comment]
For folks who'd like to use `react-basic-hooks` with 0.14, but are still waiting on the latest package set, here are some `additions` you can add to  `packages.dhall` to unblock development. I'm using my own fork of `react-basic-hooks` which contains the rebased `ps-0.14` branch and https://github.com/spicydonuts/purescript-react-basic-hooks/pull/47.

Applied these additions to the book in https://github.com/purescript-contrib/purescript-book/pull/311

```dhall
let additions =
      { react-basic =
        { dependencies = [ "prelude", "effect", "record" ]
        , repo = "https://github.com/lumihq/purescript-react-basic.git"
        , version = "main"
        }
      , react-basic-hooks =
        { dependencies =
          [ "prelude"
          , "aff-promise"
          , "aff"
          , "console"
          , "datetime"
          , "effect"
          , "either"
          , "indexed-monad"
          , "maybe"
          , "newtype"
          , "numbers"
          , "react-basic"
          , "type-equality"
          , "unsafe-coerce"
          , "unsafe-reference"
          , "web-html"
          ]
        , repo =
            "https://github.com/milesfrain/purescript-react-basic-hooks.git"
        , version = "v6.3.0-ps-0.14"
        }
      , react-basic-dom =
        { dependencies =
          [ "prelude"
          , "effect"
          , "foreign-object"
          , "react-basic"
          , "unsafe-coerce"
          , "web-dom"
          , "web-events"
          , "web-file"
          , "web-html"
          ]
        , repo = "https://github.com/lumihq/purescript-react-basic-dom.git"
        , version = "v3.2.0"
        }
      , indexed-monad =
        { dependencies = [ "control", "newtype" ]
        , repo = "https://github.com/garyb/purescript-indexed-monad.git"
        , version = "master"
        }
      }
```

**Comment by spicydonuts** [comment]
Waiting on https://github.com/nwolverson/purescript-aff-promise/pull/20 for release as well

**Comment by spicydonuts** [comment]
Released as `v7.0.0`
